src/effects/effectprocessor.h

```diff
@@ -32,7 +35,7 @@ class EffectProcessor {
     // samples. The group provided allows the effect to maintain state on a
     // per-group basis. This is important because one Effect instance may be
     // used to process the audio of multiple channels.
-    virtual void process(const QString& group,
+    virtual void process(const ChannelHandle& group,
```

Comment:
Ok, updated EffectProcessor and the native effects to have more accurate variable/function/type names and comments.